nixos/tests/all-tests.nix

```diff
@@ -887,6 +887,7 @@ in {
   systemd-homed = handleTest ./systemd-homed.nix {};
   systemtap = handleTest ./systemtap.nix {};
   tandoor-recipes = handleTest ./tandoor-recipes.nix {};
+  tandoor-recipes-script-name = handleTest ./tandoor-recipes-script-name.nix {};
```

Comment:
the commit message should be written as

```txt
nixosTests.tandoor-recipes-script-name: init
```

Besides I subscribe to ambroisie.

Reply:
Done but doesn't build